docker-compose.ci.yml

```diff
@@ -1,6 +1,28 @@
 version: '3'
 
 services:
+  test_browser:
+    build:
+      dockerfile: ./Dockerfile
+      args:
+        CI_COMMIT_SHA: ${IMAGE_TAG}
+    image: elifesciences/elife-xpub:${IMAGE_TAG}
+    # no need to start the up in any scenario, yet
+    command: sh -c "./scripts/wait-for-app.sh app:3000 ; socat -d tcp-listen:10081,reuseaddr,fork tcp:localhost:10080 & node -e \"console.log(require('config'))\";  npm run test:browser -- --screenshots /tmp/screenshots --screenshots-on-fails"
```

Comment:
is this `console.log` like, dumping some secrets in the Jenkins output?

Reply:
(I don't think so because it shouldn't have any available in the `ci` stages)

Reply:
This should be removed anyway i think

Reply:
I'd like to see if we could somehow make this switchable as its priceless when debugging the pipeline... but happy if it get relegated to documentation.

Reply:
The [output](https://alfred.elifesciences.org/blue/organizations/jenkins/pull-requests-projects%2Felife-xpub/detail/PR-1903/24/pipeline/64) doesn't have anything sensitive in it, and it shouldn't be able to as it runs code from a public repository in an isolated container. So can be left in.

Reply:
removed